modules/developer-cli-odo-installing-odo-on-vs-code.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/developer_cli_odo/installing-odo.adoc
+
+[id="installing-odo-on-vs-code_{context}"]
+
+= Installing odo on VS Code
+
+The https://marketplace.visualstudio.com/items?itemName=redhat.vscode-openshift-connector[OpenShift VS Code extension] uses both `odo` and the `oc` binary to interact with your {product title} cluster. Install the OpenShift VS Code extension on VS Code.
```

Comment:
You're missing the dash in the attribute, so it's not being properly replaced. Should be `{product-title}`.

Reply:
And the last sentence now sounds a little abrupt without the "to work with these features." Recommend adding that, or something like it, back in.

Reply:
Okay, thanks! 

Funny that the previous comment wasn't addressed...I thought I had. Thanks for catching!